src/main/java/com/sngular/kloadgen/processor/util/SchemaProcessorUtils.java

```diff
@@ -156,6 +161,12 @@ public static Descriptors.Descriptor buildProtoDescriptor(final ProtoFileElement
             schemaBuilder.addDependency(importedSchema.getFileDescriptorSet().getFile(0).getName());
             schemaBuilder.addSchema(importedSchema);
           }
+        } else {
+          final var importSchema = JMeterHelper.getParsedSchema(getSubjectName(importedClass, metadata), JMeterContextService.getContext().getProperties());
+          if (!ProtobufHelper.NOT_ACCEPTED_IMPORTS.contains(importedClass)) {
+            schemaBuilder.addDependency(((ProtobufSchema) importSchema).toDescriptor().getFullName());
+            schemaBuilder.addSchema(convertDynamicSchema((ProtobufSchema) importSchema));
```

Comment:
It is not an error, but I think it would be cleaner to create a variable to hold the value of importSchema because that way you avoid to do the cast several times.

Reply:
Fixed